tools/gbench/report.py

```diff
@@ -34,6 +36,7 @@ def __format__(self, format):
 BC_BOLD = BenchmarkColor('BOLD', '\033[1m')
 BC_UNDERLINE = BenchmarkColor('UNDERLINE', '\033[4m')
 
+UTEST_DEFAULT_ALPHA = 0.05
```

Comment:
I agree that the old hardcoding of `0.05` is wrong, but this is not correct either.
The specified `utest_alpha` (passed to `generate_difference_report()`) should be propagated/used.

It would be really great if you could submit this as a preparatory PR, to unclutter this PR :)

Reply:
> I agree that the old hardcoding of 0.05 is wrong, but this is not correct either.
> The specified utest_alpha (passed to generate_difference_report()) should be propagated/used.

You're actually referring to [this line](https://github.com/google/benchmark/blob/daa23894297781fdb1192a35d60eba1b5ad728d4/tools/gbench/report.py#L243). If so, I will make a change and send it another PR if that's what you would like.